rudder-lang/tests/techniques/supported_formats/technique.rl.cf

```diff
@@ -15,43 +15,43 @@ bundle agent supported_formats_technique {
     "Condition once_${report_data.directive_id}_0" usebundle => condition_once("cfengine_only");
 
     # Directory present:
     # 
-    #   directory("shared_cf_dsc").present() as directory_present_shared_cf_dsc
+    #   directory("""shared_cf_dsc""").present() as directory_present_shared_cf_dsc
     # 
     "Directory present_${report_data.directive_id}_1" usebundle => _method_reporting_context("Directory present", "shared_cf_dsc");
     "Directory present_${report_data.directive_id}_1" usebundle => directory_present("shared_cf_dsc");
 
-    # Directory present:
+    # directory_present:
     # 
-    #   directory("shared_cf_dsc_condition").present() as directory_present_shared_cf_dsc_condition
+    #   directory("""shared_cf_dsc_condition""").present() as directory_present_shared_cf_dsc_condition
     # 
-    "Directory present_${report_data.directive_id}_2" usebundle => _method_reporting_context("Directory present", "shared_cf_dsc_condition"),
+    "directory_present_${report_data.directive_id}_2" usebundle => _method_reporting_context("directory_present", "shared_cf_dsc_condition"),
                                              if => concat("windows");
-    "Directory present_${report_data.directive_id}_2" usebundle => directory_present("shared_cf_dsc_condition"),
+    "directory_present_${report_data.directive_id}_2" usebundle => directory_present("shared_cf_dsc_condition"),
                                              if => concat("windows");
-    "Directory present_${report_data.directive_id}_2" usebundle => _classes_noop(canonify("${class_prefix}_directory_present_shared_cf_dsc_condition")),
+    "directory_present_${report_data.directive_id}_2" usebundle => _classes_noop(canonify("${class_prefix}_directory_present_shared_cf_dsc_condition")),
                                          unless => concat("windows");
-    "Directory present_${report_data.directive_id}_2" usebundle => log_rudder("Skipping method 'Directory present' with key parameter 'shared_cf_dsc_condition' since condition 'windows' is not reached", "shared_cf_dsc_condition", canonify("${class_prefix}_directory_present_shared_cf_dsc_condition"), canonify("${class_prefix}_directory_present_shared_cf_dsc_condition"), @{args}),
+    "directory_present_${report_data.directive_id}_2" usebundle => log_rudder("Skipping method 'directory_present' with key parameter 'shared_cf_dsc_condition' since condition 'windows' is not reached", "shared_cf_dsc_condition", canonify("${class_prefix}_directory_present_shared_cf_dsc_condition"), canonify("${class_prefix}_directory_present_shared_cf_dsc_condition"), @{args}),
                                          unless => concat("windows");
 
     # Registry key present:
     # 
-    #   registry("DSC_ONLY").key_present() as registry_key_present_DSC_ONLY
+    #   registry_key("""DSC_ONLY""").present() as registry_key_present_DSC_ONLY
```

Comment:
why the triple quotes?

Reply:
i did not touched that file ... i don't know how where they modified